### ISSUE TITLE
No Channels Working

### ISSUE THREAD

**Issue by Nzzyme** [botallen/repository.botallen#330]
None of the channels are working today. Tried clearing cache and re login. Anyone else facing this issue?

**Comment by Aeropro4** [comment]
Yeah me too

**Comment by Aeropro4** [comment]
I think it's the end for botallen

**Comment by rajeevCs** [comment]
Same issue ,you guys can try jio tv plus ,you can google jio tv plus apk for Android TV and install it on tv ,some channels works there

**Comment by ajay7413** [comment]
> None of the channels are working today. Tried clearing cache and re login. Anyone else facing this issue?

Yes Me Too

**Comment by ajay7413** [comment]
Yes Me Too

**Comment by echakkara** [comment]
jio tv plus apk seems to wrok for less channels only. But Bot Allen Kodi add-on never disappointed and it played TV channels like a champ for about 2 years

**Comment by Aeropro4** [comment]
IT'S FIXED GUYS

**Comment by LogicalError007** [comment]
I'll try changing DNS. Seems like it's being blocked.

**Comment by Aeropro4** [comment]
> I'll try changing DNS. Seems like it's being blocked.

So did it work?

**Comment by LogicalError007** [comment]
> So did it work?
On mobile it worked partially. I'll try this method on fire stick after college.

**Comment by technofreak24** [comment]
Colors Rishtey, Colors Marathi SD/HD, SET HD, SAB HD, Sony Pal, Sony Wah are working channels after two-three taps on FireTV Stick App Remote. Other channels are not loading at all.
Screenshots at https://github.com/botallen/repository.botallen/issues/335#issuecomment-1368438173

**Comment by adityapv** [comment]
Let's hope @iMro0t fixes the issue in his spare time. In the meanwhile, you can use Samsung TV Plus addon from [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository and [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) (for zee5 (has live channels), sonyliv (has live channels), voot (has live channels), hotstar and mxplayer)

**Comment by Aeropro4** [comment]
> Let's hope @iMro0t fixes the issue in his spare time. In the meanwhile, you can use Samsung TV Plus addon from [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository and [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) (for zee5 (has live channels), sonyliv (has live channels), voot (has live channels), hotstar and mxplayer)

Sorry to say but both the addons are not that great or useless
The [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository has only few and free channels and are kinda hard to access many clicks to get there etc.
And the [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) repository needs login for every addon and for login we need subscription

**Comment by adityapv** [comment]
> > Let's hope @iMro0t fixes the issue in his spare time. In the meanwhile, you can use Samsung TV Plus addon from [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository and [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) (for zee5 (has live channels), sonyliv (has live channels), voot (has live channels), hotstar and mxplayer)
> 
> Sorry to say but both the addons are not that great or useless
> The [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository has only few and free channels and are kinda hard to access many clicks to get there etc.
> And the [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) repository needs login for every addon and for login we need subscription 

Yes, slyguy has lot of addons that are not quite good, but for me, I use India channels from Samsung TV Plus, also in Ajayk Repo, you can use sonyliv (live channels too) and zee5 without login only voot and hotstar requires subscription to access content. You can mark those channels as favorites and access them all under favorites tab. You can also use [OrderFavourites](https://github.com/doko-desuka/plugin.program.orderfavourites) to organize favorites.

**Comment by adityapv** [comment]
> > > Let's hope @iMro0t fixes the issue in his spare time. In the meanwhile, you can use Samsung TV Plus addon from [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository and [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) (for zee5 (has live channels), sonyliv (has live channels), voot (has live channels), hotstar and mxplayer)
> > 
> > Sorry to say but both the addons are not that great or useless
> > The [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository has only few and free channels and are kinda hard to access many clicks to get there etc.
> > And the [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) repository needs login for every addon and for login we need subscription 
> 
> Yes, slyguy has lot of addons that are not quite good, but for me, I use India channels from Samsung TV Plus, also in Ajayk Repo, you can use sonyliv (live channels too) and zee5 (free live channels only: mostly news) without login. voot and hotstar requires subscription to access content. You can mark those channels as favorites and access them all under favorites tab. You can also use [OrderFavourites](https://github.com/doko-desuka/plugin.program.orderfavourites) to organize favorites.

**Comment by Aeropro4** [comment]
> > > Let's hope @iMro0t fixes the issue in his spare time. In the meanwhile, you can use Samsung TV Plus addon from [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository and [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) (for zee5 (has live channels), sonyliv (has live channels), voot (has live channels), hotstar and mxplayer)
> > 
> > Sorry to say but both the addons are not that great or useless
> > The [slyguy](https://www.matthuisman.nz/2020/02/slyguy-kodi-repository.html?m=1) repository has only few and free channels and are kinda hard to access many clicks to get there etc.
> > And the [AjayKRepo](https://github.com/Vikassm73/AjaykRepo) repository needs login for every addon and for login we need subscription 
> 
> Yes, slyguy has lot of addons that are not quite good, but for me, I use India channels from Samsung TV Plus, also in Ajayk Repo, you can use sonyliv (live channels too) and zee5 (free live channels only: mostly news)without login. voot and hotstar requires subscription to access content. You can mark those channels as favorites and access them all under favorites tab. You can also use [OrderFavourites](https://github.com/doko-desuka/plugin.program.orderfavourites) to organize favorites.

Hmm can't argue with it ,have no choice aswell

**Comment by AMEET82** [comment]
Same problem
Some channels are working.
But after a while frozen.
So many channels are failing to play.
Plz solve.

**Comment by Nzzyme** [comment]
I found a workaround guys but sadly it only works with Jio network. That is, you will need to use Jio mobile hotspot on your Android TV for this to work. It doesn't work with other ISP

Install the modded Jio TV app for Android TV from [here](https://forum.xda-developers.com/attachments/jiotv_5-9-5-tv-apk.5118493/)

This is a mod available from [this](https://forum.xda-developers.com/t/app-mod-cast-jiotv-mod-with-miracast-android-tv-and-firestick-support.3857654/) page

Login with your Jio credentials and you should be able to play all Jio channels.

**Comment by adityapv** [comment]
Seems like issue has been fixed in this one: https://github.com/Geletinousamigo/plugin.video.jiotv

**Comment by Nzzyme** [comment]
> Seems like issue has been fixed in this one: https://github.com/Geletinousamigo/plugin.video.jiotv

I just tried installing the zip file in that other repo and it failed to install. Does it work for you?


![Untitled](https://user-images.githubusercontent.com/6959116/211139647-b38f4fae-6f28-4bfd-ac7c-27a26dedbf90.jpg)
